### ISSUE TITLE
document and implement missing specs

### ISSUE THREAD

**Issue by gidden** [cyclus/benchmarks#12]
In writing the NEA and INPRO specifications, I noticed that both described reactor cycle lengths using EFPD and capacity factors. Accordingly, that should be added to the spec instead of using months, for now -- I can also provide this as an example of something that's Cyclus specific (using month time steps). Additionally, I need to implement the Enrichment Translator, which should be a good example of how reusable my current translation implementation is.

In short: 
- describe and implement cycle length as EFPD \* CF
- implement Enrichment translator
